.github/CODEOWNERS

```diff
@@ -669,7 +669,6 @@
 
 # ServiceLabel: %Network - CDN
```

Comment:
If this is intended to be the management library ownership, it will need to move into the management section around L930 and also have a `%Mgmt` entry for the `ServiceLabel`.    As this exists today, it will be treated as a data plane library label for triage.